src/SFA.DAS.Apprenticeships.Api.Types/V3/ApprenticeshipAutocompleteSearchResultsItem.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace SFA.DAS.Apprenticeships.Api.Types.V3
+{
+    public class ApprenticeshipAutocompleteSearchResultsItem
```

Comment:
Should this be named just SearchResultItem?  The fact that it's used to autocomplete a field is just a UI concern.